cms/models/pluginmodel.py

```diff
@@ -250,6 +250,23 @@ def get_plugin_instance(self, admin=None):
         self._inst = instance
         return self._inst, plugin
 
+    def get_bound_plugin(self, admin=None):
```

Comment:
`get_plugin_instance()` unfortunately returns plugin instance and plugin admin instance which I dislike as it forces us to always do `get_plugin_instance()[0]`

Instead, as I've started to introduce the terminology of bound and unbound plugins, I decided to create this more explicit method which will only return the plugin instance.

That said, you found a small bug.. I need to remove the `admin` parameter :man_facepalming: